src/skin/popup.html

```diff
@@ -80,5 +80,6 @@ <h1 id="report_title" class="i18n_report_title"></h1>
   <center><button id="donate" class="pbButton" style="display:block" target="blank"><span class="i18n_donate_to_eff"></span></button></center>
 </div>
 <div class='clear'></div>
+<div class='footer'><span id="version" class="i18n_version"></span></div>
```

Comment:
Upon restarting my browser, the localization string "version" started being displayed. I guess it was working all along, but wasn't updating locally for me for some reason.

Regarding your suggested changes, I reduced the size of "Privacy Badger" and moved it up a bit, and aligned the version string along the left edge. I've only used CSS and handful of times before and couldn't figure out how to move the version string up. I tried the same technique I used to move "Privacy Badger" up (adjusting the margin), but it seems to be stuck at this height and won't go any higher:

![pb_css](https://cloud.githubusercontent.com/assets/4513790/25594732/b787da9e-2e76-11e7-89f1-7ed0f5ba3853.png)